packages/wonder-blocks-form/src/components/field-heading.stories.js

```diff
@@ -0,0 +1,114 @@
+// @flow
+import * as React from "react";
+
+import {TextField} from "@khanacademy/wonder-blocks-form";
+import {LabelLarge, LabelSmall} from "@khanacademy/wonder-blocks-typography";
+
+import type {StoryComponentType} from "@storybook/react";
+import FieldHeading from "./field-heading.js";
+
+export default {
+    title: "Form / FieldHeading",
+};
+
+export const basic: StoryComponentType = () => {
+    const [value, setValue] = React.useState("khan");
+    const [errorMessage, setErrorMessage] = React.useState();
+    const [focused, setFocused] = React.useState(false);
+    const uniqueId = "uniqueid123";
+
+    const handleOnChange = (newValue: string) => {
+        setValue(newValue);
+    };
+
+    const validation = (value: string) => {
+        const emailRegex = /^[^@\s]+@[^@\s.]+\.[^@.\s]+$/;
+        if (!emailRegex.test(value)) {
+            return "Please enter a valid email";
+        }
+    };
+
+    const handleOnValidation = (errorMessage: ?string) => {
+        setErrorMessage(errorMessage);
+    };
+
+    const handleOnKeyDown = (
+        event: SyntheticKeyboardEvent<HTMLInputElement>,
+    ) => {
+        if (event.key === "Enter") {
+            event.currentTarget.blur();
+        }
+    };
+
+    const handleOnFocus = () => {
+        setFocused(true);
+    };
+
+    const handleOnBlur = () => {
+        setFocused(false);
+    };
+
+    return (
+        <FieldHeading
+            field={
+                <TextField
+                    id={`${uniqueId}-field`}
+                    aria-describedby={`${uniqueId}-error`}
+                    aria-invalid={errorMessage ? "true" : "false"}
```

Comment:
Great, thanks for adding the examples here! Looking forward to see this implementation in `LabeledTextField`.